### DISCUSSION TITLE
Listening to Slash Commands

### DISCUSSION THREAD

**Discussion by Mehul343** [nextcord/nextcord#510] [Q&A] [answered]
How can I listen to slash commands?

**Comment by ooliver1** [comment, marked as answer]
on_interaction may be what you want

**Reply by Mehul343** [reply]
> on_interaction may be what you want

I tried but it doesn't eem to do anything. Can you show me an example?

**Reply by techpluscraft** [reply]
https://nextcord.readthedocs.io/en/stable/api.html?highlight=on_interaction#nextcord.on_interaction
